plugins/@grouparoo/pipedrive/src/lib/parallelism.ts

```diff
@@ -0,0 +1,24 @@
+import { AppParallelismMethod, objectCache } from "@grouparoo/core";
+
+const DEFAULT_VALUE = 10;
+
+export const parallelism: AppParallelismMethod = async ({
+  app,
+  appOptions,
+}) => {
+  const cachedValue = await objectCache(
```

Comment:
Let's move the read of this value to the PipedriveClient, so it's more likely to share the same key and access method.

Reply:
But the `parallelism` method doesn't receive client instance as one of its parameters!
How could we call it from there?